### PR TITLE
src/components: add section registration details to ProfileDetails component

### PR DESCRIPTION
**Pull request by maceksimon**
Add section registration details to `ProfileDetails` component:

* Add logic for displaying icon and label
* Add translations
* Add tests

### CODE REVIEW

File: src/components/__tests__/ProfileDetails.cy.js [hunk 1]
```diff
@@ -457,4 +469,52 @@ function coreTests() {
...
+        .should('be.visible')
+        .and('have.css', 'font-size', '20px')
+        .and('have.css', 'font-weight', '500')
+        .and('have.color', grey10)
+        .and('contain', i18n.global.t('profile.titleRegistrationDetails'));
+      // payment state
+      cy.dataCy(selectorPaymentState)
+        .find(dataSelectorLabel)
+        .should('be.visible');
+      if (formPersonalDetails.paymentState === 'paid') {
```
Comment (tmszi) [inline src/components/__tests__/ProfileDetails.cy.js:486]: Use `paymentState` TS enum please.

File: src/components/profile/ProfileDetails.vue [hunk 2]
```diff
@@ -50,8 +56,41 @@ export default defineComponent({
...
+
+    const iconPaymentColor = computed(() => {
+      return profile.paymentState === PaymentState.paid ||
+        profile.paymentState === PaymentState.paidByOrganization
+        ? 'green'
+        : 'red';
+    });
+
+    const iconPaymentState = computed(() => {
+      return profile.paymentState === PaymentState.paid ||
```
Comment (tmszi) [inline src/components/profile/ProfileDetails.vue:78]: Alternatively you can use something like this:

 ```ts
`[PaymentState.paid, PaymentState.paidByOrganization].includes(profile.paymentState)
```

File: src/components/profile/ProfileDetails.vue [hunk 3]
```diff
@@ -260,5 +297,56 @@ export default defineComponent({
...
+          :value="profile.package.title"
+          class="col-12 col-md-6 items-center"
+          data-cy="profile-details-payment-state"
+        >
+          <template #value>
+            <div class="row q-col-gutter-md justify-between">
+              <div class="col-12 col-sm-auto flex items-center gap-8">
+                <q-icon
+                  :name="iconPaymentState"
+                  size="18px"
```
Comment (tmszi) [inline src/components/profile/ProfileDetails.vue:323]: `18px` repeat, you can use `const`, please.

Comment (maceksimon) [reply src/components/profile/ProfileDetails.vue]: Updated using const.

File: src/components/profile/ProfileDetails.vue (hunk 2 above)
Comment (maceksimon) [reply src/components/profile/ProfileDetails.vue]: Refactored using includes.

**Pull request merged**